### DISCUSSION TITLE
공식문서

### DISCUSSION THREAD

**Discussion by InChil2** [InChil2/SlotTagging#12] [참고자료]
[Message 플랫폼 사용설명 공식 문서](https://developers.facebook.com/docs/messenger-platform/send-messages/?translation#send_api_basics)

**Comment by InChil2** [comment]
[한국 국립국어원 말뭉치](https://corpus.korean.go.kr/request/corpusRegist.do)

**Comment by InChil2** [comment]
[ETRI KoBERT](https://aiopen.etri.re.kr/service_dataset.php)

**Comment by sumunoh** [comment]
[burt 이론 및 과정 설명](https://docs.likejazz.com/bert/)

**Comment by InChil2** [comment]
[SKTBrain KoBERT](https://github.com/SKTBrain/KoBERT)
